### PR TITLE
Login and registration

### PR DESCRIPTION
**Pull request by vakyl9r**
Make cute log_in and registration pages with devise gem and bootstrap

### CODE REVIEW

**Comment by vakyl9r** [comment]
Not much, but does the job done))

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -1,89 +1,18 @@
...
-td {
-  padding-bottom: 7px;
-  padding-left: 5px;
-  padding-right: 5px;
-}
-
-div {
-  &.field, &.actions {
-    margin-bottom: 10px;
+.registration_block{
```
Comment (Magons) [inline app/assets/stylesheets/scaffolds.scss]: leave a space before the bracket

File: app/assets/stylesheets/scaffolds.scss [hunk 2]
```diff
@@ -1,89 +1,18 @@
...
+  margin-top:50px;
+  border-color: black;
+  border-width: thin;
+  border-style: solid;;
+  border-radius: 5px;
+  width: 50%;
+  padding: 20px;
+  margin-left:auto;
+  margin-right:auto;
+  .field{
```
Comment (Magons) [inline app/assets/stylesheets/scaffolds.scss]: the same

File: app/assets/stylesheets/scaffolds.scss [hunk 3]
```diff
@@ -1,89 +1,18 @@
...
+    margin-top: 5px;
+    input{
```
Comment (Magons) [inline app/assets/stylesheets/scaffolds.scss]: the same

File: app/views/devise/registrations/new.html.haml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.registration_block
+  %h2 Sign up
```
Comment (Magons) [inline app/views/devise/registrations/new.html.haml]: better use ruby string
`%h2= 'Sign up'`

**Review by Magons — COMMENTED** [review]
correct my remarks and you can merge

File: app/views/devise/registrations/new.html.haml (hunk 4 above)
Comment (vakyl9r) [reply app/views/devise/registrations/new.html.haml]: okay

**Pull request merged**